View/media_control_panel.py

```diff
@@ -11,3 +11,17 @@ def __init__(self):
         to the panel using some layout.
         """
         super().__init__()
+        
+        # create play pause btn
+        self.playBtn = QPushButton()
+        self.playBtn.setIcon(self.style().standardIcon(QStyle.SP_MediaPlay))
+        #create horizontal box layout
+        hboxLayout = QHBoxLayout()
+        hboxLayout.setContentsMargins(0, 0, 0, 0)
+        hboxLayout.addWidget(self.playBtn)
+
+        #add vertical box layout to add all widgets
```

Comment:
What is the purpose of the vertical layout box? Can't we just set the layout of the widget to the horizontal layout?